src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -49,12 +50,17 @@ internal sealed class TracerProviderSdk : TracerProvider
                 Debug.Assert(this.OwnedServiceProvider != null, "serviceProvider was not IDisposable");
             }
 
+            OpenTelemetrySdkEventSource.Log.TracerProviderSdkEvent("Building TracerProvider.");
+
             var state = new TracerProviderBuilderState(serviceProvider);
 
             TracerProviderBuilderServiceCollectionHelper.InvokeRegisteredConfigureStateCallbacks(
                 serviceProvider,
                 state);
 
+            StringBuilder processorsAdded = new StringBuilder();
```

Comment:
please also include sampler, as it is often the most critical information.

Reply:
@cijothomas sure, will raise another PR for this